studio/plugins/dj-order/index.tsx

```diff
@@ -1,213 +1,213 @@
 // /plugins/dj-order/index.tsx
```

Comment:
This seems like a candidate for decomposition into smaller modules. We should also add tests for them.